Documentation/Update/Index.rst

```diff
@@ -0,0 +1,128 @@
+.. include:: ../Includes.txt
+
+.. highlight:: bash
+
+.. _update:
+
+==================
+TYPO3 Patch Update
+==================
+
+
+.. _typo3-version-basics:
+
+TYPO3 Version Basics
+====================
+
+The TYPO3 version numbers consist of the following parts:
+
+<major>.<minor>.<patch>
+
+For example 9.5.1 is major version 9, minor version 5 and patch version 1.
+
+* **major**: An incrementation in a major version (e.g. 8.7.23 -> 9.5.5) may mean breaking
+  changes and requires an extensive update procedure.
+* **minor**: A change in a minor version should contain no deprecation. This means the minor
+  version is backwards compatible (e.g. 6.0.0 -> 6.2.0). The new minor version may
```

Comment:
This isn't true, at least until v10 as far as I know.

TYPO3 is not using semantiv versioning. Especially v6.x introduced breaking changes, same goes for 7.x.

Since v9.x the main goal is to introduce breaking changes with vx.0.0 and keep compatible. Then TYPO3 will release vx.1.0 vx.2.0 etc. Which does not mean anything about compatibility in the first place.